### DISCUSSION TITLE
Apache Commons Text的漏洞

### DISCUSSION THREAD

**Discussion by LCehoennardo** [apache/doris#13483] [A - General / Q&A] [answered]
最近Apache Commons Text发现了一个[vulnerability](https://nvd.nist.gov/vuln/detail/CVE-2022-42889)，doris fe使用到了checker-3.22.2.jar包，checker中用到了Apache Commons Text
> Apaches commons text is a dependency of plume-util:options. Options is a transitive dependency of the Checker Framework via Annotation File Utilities (AFU). 

请问一下这个漏洞什么时候会修复呢，或者我们可以通过什么方式自行修复？

**Comment by dataalive** [comment]
这个我们来研究下

**Comment by hf200012** [comment, marked as answer]
https://github.com/apache/doris/pull/13562/files

**Reply by LCehoennardo** [reply]
Thank you
